stunnel/stunnel.mli

```diff
@@ -53,10 +54,15 @@ val connect :
   string -> int -> t
 
 (** Disconnects from stunnel and cleans up *)
-val disconnect : ?wait:bool -> ?force:bool -> t -> unit
+val disconnect : ?wait:bool -> ?force:bool -> ?verbose:bool -> t -> unit
 
 val diagnose_failure : t -> unit
 
 val test : string -> int -> unit
 
 val must_verify_cert : bool option -> bool
+
+val config_file : bool -> bool -> string -> int -> bool -> string
```

Comment:
Exposing this function seems a bit of a leak. Particularly since it is just for logging and only logging an example (dummy) config.